docs/spark-procedures.md

```diff
@@ -271,7 +271,7 @@ Iceberg can compact data files in parallel using Spark with the `rewriteDataFile
 |---------------|-----------|------|-------------|
 | `table`       | ✔️  | string | Name of the table to update |
 | `strategy`    |    | string | Name of the strategy - binpack or sort. Defaults to binpack strategy |
-| `sort_order`  |    | string | If Zorder, then comma separated column names within zorder() text. Example: zorder(c1,c2,c3). <br/>Else, Comma separated sort_order_column. Where sort_order_column is a space separated sort order info per column (ColumnName SortDirection NullOrder). <br/> SortDirection can be ASC or DESC. NullOrder can be NULLS FIRST or NULLS LAST |
+| `sort_order`  |    | string | If Zorder(supported in Spark 3.2 and above), then comma separated column names within zorder() text. Example: zorder(c1,c2,c3). <br/>Else, Comma separated sort_order_column. Where sort_order_column is a space separated sort order info per column (ColumnName SortDirection NullOrder). <br/> SortDirection can be ASC or DESC. NullOrder can be NULLS FIRST or NULLS LAST |
```

Comment:
One more comment here, instead of "if Zorder", maybe "For Zorder use a comma separated list of columns within zorder(). (Supported in Spark 3.2 and Above) Example: .... "

"Else, Comma separated sort orders in the format (ColumnName SortDirection NullOrder) where .... 
Defaults to the table's sort order."

Just reading over the whole doc it is a little confusing